plugins/view-resources/src/components/ObjectPresenter.svelte

```diff
@@ -50,6 +50,7 @@
 
   $: if (doc !== undefined) {
     getObjectPresenter(client, doc._class, { key: '' }).then((p) => {
+      console.log(`PRESENTER: ${p._class}, ${p.presenter.name}`)
```

Comment:
I do not think we need this logged.

Reply:
Sorry, I forgot to remove debugging logs. Removed it now, thanks